tests/suit/framework/ova.go

```diff
@@ -17,7 +17,7 @@ func (r *OvaClient) LoadSourceDetails() (vm *OvaVM, err error) {
 		r.storageClass = "nfs-csi"
 	}
 
-	r.vmData.testVMId = "c5686650854d1e69b4123f4bf2e70fe1ed2a"
+	r.vmData.testVMId = "3920ddbed8bb43f1bd6e00e412635f99727d"
```

Comment:
Note: Once the OVA VM struct changes it changes the database ID and we need to update it here.

We should fix it in a more dynamic way.